libtiledbsoma/test/common.cc

```diff
@@ -153,7 +153,7 @@ std::unique_ptr<ArrowSchema> create_index_cols_info_schema(
 
     auto schema = ArrowAdapter::make_arrow_schema(names, tiledb_datatypes);
 
-    for (size_t i = 0; i < schema->n_children; ++i) {
+    for (size_t i = 0; i < (size_t)schema->n_children; ++i) {
```

Comment:
I don't mean to nitpick, but:
* do we encourage use of C-style (legancy) casts, vs say a `static_cast<>`?
* if `schema->n_children` is signed, alternative might just be to make `i` signed?
either works for me...

Reply:
Thanks @bkmartinjr !